snippets/language-ember-htmlbars.cson

```diff
@@ -1 +1,28 @@
-
+'.text.html.htmlbars':
+  'if...end':
+    'prefix': 'if'
+    'body': '{{#if ${1:condition}}}\n\t$0\n{{/if}}'
+  'if...else...end':
+    'prefix': 'ife'
+    'body': '{{#if ${1:condition}}}\n\t$0\n{{else}}\n\t\n{{/if}}'
+  'unless':
+    'prefix': 'unl'
+    'body': '{{#unless ${1:condition}}}\n\t$0\n{{/unless}}'
+  'ember-action':
+    'prefix': 'ema'
+    'body': '{{action ${1:\'${2:name}\'}}}'
+  'inline-if':
+    'prefix': 'ifl'
+    'body': '{{if ${1:condition} \'${2:property}\'}'
+  'link-to':
+    'prefix': 'lt'
+    'body': '{{#link-to \'${route}\'}}\t$0{{/link-to}}'
+  'each':
+    'prefix': 'each'
+    'body': '{{#each ${1:model} key="@guid" as |${2:param}|}}\n\t$0\n{{/each}}'
```

Comment:
Do you mind removing key="@guid"? [This PR](https://github.com/emberjs/ember.js/pull/11461) has made it redundant.